### ISSUE TITLE
Github broke the UI again

### ISSUE THREAD

**Issue by MrNaif2018** [Kir-Antipov/GitHub-Defreshed#49]
## Description

Today I opened the repository page, and it got stretched in an unnatural way. Reproducable both in chrome and firefox

## Screenshot

![изображение](https://user-images.githubusercontent.com/39452697/158795717-4b97c29a-d2b4-48bd-899a-306a875b5f61.png)
![изображение](https://user-images.githubusercontent.com/39452697/158795745-0f684390-462c-4c87-81ca-fb2598608442.png)

## Other information

- **Link to page with the issue**: https://github.com/Kir-Antipov/GitHub-Defreshed
- **Browser**:  Firefox/Chrome latest
- **Operating System**: Linux Mint 20.3 XFCE
- **GitHub-Defreshed version**: 4.4.1

**Comment by Kira-NT** [comment]
Oh no, why does everything have to break during the goddamn war?

Can confirm, but I don't have any mental capacity to work on something at the moment, so pr will be really appreciated

**Comment by MrNaif2018** [comment]
Yeah, I can totally understand you ):
I may try to check it, is it somewhere near https://github.com/Kir-Antipov/GitHub-Defreshed/blob/master/src/fixers/repository/columns-fixer.ts?

**Comment by Kira-NT** [comment]
I think it's more about [`ContainerFixer`](https://github.com/Kir-Antipov/GitHub-Defreshed/blob/master/src/fixers/repository/container-fixer.ts)

**Comment by medievil1** [comment]
PR, issue pages are normal width, so likely github made a change to allow the front page to use full resolution width instead of a fixed scalable size.. using browser zoom - side only shrinks the font, the page container does not change... it looks a bit weird, BUT everything is where it is supposed to be. just text is dangerously close to the edge on the left and could get cut off

**Comment by MrNaif2018** [comment]
To update: even with #50 (using it for a while now, thanks @An-dz), today when loading repository page, then clicking for example pull requests button and then home button again, the UI is a bit broken once again

![изображение](https://user-images.githubusercontent.com/39452697/164533414-e2c76768-ee7c-486b-8bd1-468f54dbfc01.png)

**Comment by An-dz** [comment]
@MrNaif2018 What do you mean by _"Home button"_?

**Comment by MrNaif2018** [comment]
I meant code button to return to repository's main page, sorry

**Comment by An-dz** [comment]
Managed to reproduce now, but won't be able to fix it anytime soon.

**Comment by medievil1** [comment]
it is broken again unfortunately
![image](https://github.com/Kir-Antipov/GitHub-Defreshed/assets/38597905/7c5e8079-011f-4de6-8b83-2909c390c763)

**Comment by Kira-NT** [comment]
Yeah, it's been broken for quite a while now. Sadly, as a single maintainer, I cannot keep up with all the updates that are constantly being pushed by GitHub. While many of them are subtle and often don't even change anything visually, they do alter the layout and class names I relied on to make this project work.

Don't get me wrong, I still dislike the "refreshed" UI; not because of the round buttons and stuff, but because it's simply less practical than the original UI. However, I've decided to accept defeat. Pouring as much time and effort into this project, as it needs to continue functioning, is even more impractical than just dealing with the new UI.
